lib/purserl.ex

```diff
@@ -234,21 +299,21 @@ defmodule DevHelpers.Purserl do
         case Jason.decode(msg) do
           {:ok, v} ->
             # yes, now do stuff with it
-            process_warnings(state, start_compile_at, v["warnings"], v["errors"], :wip)
+            process_warnings(state, DateTime.utc_now(), v["warnings"], v["errors"], :wip)
```

Comment:
This looks race-y. Does this improve the errors anything?

Reply:
It doesn't really change anything actually. The `start_compile_at` variable did not actually contain the datetime when we started compiling, it was updated for each new message. This only made it confusing imo.

Reply:
Comment code please. We'll ask that same question again later

Reply:
Could we change this to `state.started_at` instead then? That might improve error caching somewhat! :D